tests/framework/pca_postprocessor/gold/transform/test_importantRank_Dataset_dump.xml

```diff
@@ -1,25 +0,0 @@
-<DataObjectMetadata name="test_importantRank_Dataset">
-  <DataSet type="Static">
-    <dims>
-      <importanceIndex_x_y1>time</importanceIndex_x_y1>
-      <sensitivityIndex_z_y1>time</sensitivityIndex_z_y1>
-      <sensitivityIndex_y_x1>time</sensitivityIndex_y_x1>
-      <sensitivityIndex_y_y1>time</sensitivityIndex_y_y1>
-      <importanceIndex_x_x1>time</importanceIndex_x_x1>
-      <importanceIndex_y_x1>time</importanceIndex_y_x1>
-      <sensitivityIndex_x_y1>time</sensitivityIndex_x_y1>
-      <importanceIndex_z_x1>time</importanceIndex_z_x1>
-      <importanceIndex_z_y1>time</importanceIndex_z_y1>
-      <importanceIndex_y_y1>time</importanceIndex_y_y1>
-      <pcaIndex_x1>time</pcaIndex_x1>
-      <pcaIndex_y1>time</pcaIndex_y1>
-      <sensitivityIndex_z_x1>time</sensitivityIndex_z_x1>
-      <sensitivityIndex_x_x1>time</sensitivityIndex_x_x1>
-    </dims>
-    <general>
-      <outputs>importanceIndex_x_y1,importanceIndex_x_x1,importanceIndex_y_y1,importanceIndex_y_x1,importanceIndex_z_y1,importanceIndex_z_x1,sensitivityIndex_x_y1,sensitivityIndex_x_x1,sensitivityIndex_y_y1,sensitivityIndex_y_x1,sensitivityIndex_z_y1,sensitivityIndex_z_x1,pcaIndex_y1,pcaIndex_x1</outputs>
-      <sampleTag>RAVEN_sample_ID</sampleTag>
-    </general>
-  </DataSet>
-  
-</DataObjectMetadata>
```

Comment:
Is this worth testing for its output?  Or is it tested somewhere else?

Reply:
I think the csv file should be enough.